src/item.cpp

```diff
@@ -1399,6 +1399,13 @@ std::string item::info( bool showtext, std::vector<iteminfo> &info ) const
         }
 
         if( !magazine_integral() ) {
+            const auto cur_mag = magazine_current();
```

Comment:
I went with a different one because magazine sounds weird here.
Say, battery in a radio would be a "magazine".